docs/troubleshooting-guide/cant-log-in.md

```diff
@@ -0,0 +1,95 @@
+# People can't log in to Metabase
+
+You should be able to log in to Metabase, but:
+
+- you can't see the login page, or
+- your credentials aren't accepted.
+
+## Are you using the right URL for your Metabase?
+
+**Root cause:** the Metabase instance you are trying to log in to isn't where you think it is or isn't accessible.
+
+**Steps to take:**
+
+1. Check whether you need to include a port number as well as a hostname in the connection URL. For example, Metabase might be at `https://example.com:3000/` instead of `https://example.com/`.
+   - If you're an administrator, you'll have configured this.
+   - If you're not, please ask your admin.
+2. Check whether your Metabase instance has moved. For example, if you were using a trial instance of Metabase, but you're now in production, the URL might have changed.
+
+## Does your Metabase use Single Sign-On (SS0) to manage accounts?
+
+Metabase can manage accounts itself, or administrators can configure it to let people sign on using third-party credentials. You can tell what kind of account you have when you try to sign in: if your login ID is your email address, Metabase is managing the account, but if you're using the option to sign in which a third-party identity provider, someone else is responsible for authenticating you.
```

Comment:
I would remove this. It's also not correct - LDAP would be an example.

Reply:
Hm - @jeff-bruemmer thought this was OK. As a regular user, how can I tell if my credentials are in Metabase or via SSO?

Reply:
The user would not know that unless they login. And the admin cannot really see that either depending on how the user was initially created.

Reply:
We should chat...